types/v1/types.go

```diff
@@ -122,6 +101,16 @@ type Result struct {
 	Plugin string `json:"plugin"`
 	// Version of the plugin
 	Version string `json:"version"`
+	// error details if IsError is set
+	ErrorMessage string `json:"errormessage"`
```

Comment:
I'd probably just change this to `Error string `json:"error"``

Reply:
done